app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationList.jsx

```diff
@@ -525,12 +541,14 @@ export default function RegistrationAdministrationList({ competitionInfo }) {
 }
 
 function SectionToggle({
-  icon, title, inParens, color,
+  icon, title, inParens, color, sectionRef,
 }) {
   return (
-    <Header as="span" size="large">
-      <Icon name={icon} color={color} />
-      {`${title} (${inParens})`}
-    </Header>
+    <span ref={sectionRef} style={{ scrollMarginTop: '7em' }}>
```

Comment:
Does the `Header` component itself not support `ref` natively? If not, I think SemUI has a `Ref` component that you can use to pass refs down within their own framework

Reply:
(we do actually use that component in a couple of places already, so just search for `<Ref` throughout our codebase and you will see)